target/linux/qualcommax/image/ipq807x.mk

```diff
@@ -247,6 +247,19 @@ define Device/redmi_ax6
 endef
 TARGET_DEVICES += redmi_ax6
 
+define Device/spectrum_sax1v1k
+	$(call Device/FitImage)
+	$(call Device/EmmcImage)
+	DEVICE_VENDOR := Spectrum
+	DEVICE_MODEL := SAX1V1K
+	DEVICE_DTS_CONFIG := config@rt5010w-d187-rev6
+	BLOCKSIZE := 128k
+	PAGESIZE := 2048
```

Comment:
`BLOCKSIZE` and `PAGESIZE` are not used in the case of MMC.

Reply:
fixed